modules/cnf-image-based-upgrade-seed-image-config.adoc

```diff
@@ -140,6 +144,40 @@ a|`SriovNetwork.yaml`
 `SriovNetworkNodePolicy.yaml`
 |If the configuration, including namespaces, is exactly the same on both the seed and target cluster, you can include them in the seed image. Otherwise, apply them as extra manifests.
 |====
+endif::[]
+
+ifdef::ibi[]
+The following table lists resources and configurations that you must apply as extra manifests or by using {rh-rhacm} policies:
+
+.Additional resources for image-based installations
```

Comment:
I think technically all these resource be enabled via extra manifests or RHACM policies but for the GitOps flow they should be enabled by policies so they can be managed later, e.g. apply configuration changes.